pkg/network/vmispec/interface_test.go

```diff
@@ -94,15 +94,6 @@ var _ = Describe("VMI network spec", func() {
 
 	const iface1, iface2, iface3, iface4, iface5 = "iface1", "iface2", "iface3", "iface4", "iface5"
 
-	DescribeTable("return VMI spec interface names, given",
```

Comment:
Why was this test removed?

Reply:
It was a mistake having it in the first place because there is no logic to test, all InterfaceName() does is indexing the interface since by name.

Reply:
This function is removed altogether in favor of using a helper that already exists.